HomeHealthServices/R4/resources/Questionnaire-R4-HomeHealthServices.json

```diff
@@ -30,34 +66,660 @@
+          "type": "string",
+          "required": false,
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HomeHealthServicesPrepopulation\".XXXXXXXXXXXXXXX_duration"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "6",
+      "text": "Justification and signature if the patient’s sole skilled service need is for skilled oversight of unskilled services (management and evaluation of the care plan or complex care plan management):",
+      "type": "group",
+      "item": [
+        {
+          "linkId": "6.1",
+          "text": "Justification",
+          "type": "text",
+          "required": false,
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HomeHealthServicesPrepopulation\".OrdersJustification"
+              }
+            }
+          ]
+        },
+        {
+          "linkId": "6.2",
+          "text": "Signature",
+          "type": "string",
+          "required": true,
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HomeHealthServicesPrepopulation\".OrdersJustificationSignature"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "7",
+      "text": "Physical Therapy Services",
+      "type": "group",
+      "repeats": true,
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+          "valueCodeableConcept": {
+            "coding": [
+              {
+                "system": "http://hl7.org/fhir/questionnaire-item-control",
+                "code": "gtable"
+              }
+            ]
+          }
+        }
+      ],
+      "item": [
+        {
+          "linkId": "7.1",
+          "text": "Service Name",
+          "type": "open-choice",
+          "required": true,
+          "answerValueSet": "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1219.99",
+          "extension": [
+            {}
+          ]
+        },
+        {
+          "linkId": "7.2",
+          "text": "Frequency",
+          "type": "choice",
+          "required": false,
+          "answerValueSet": "http://hl7.org/fhir/ValueSet/timing-abbreviation",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"HomeHealthServicesPrepopulation\".XXXXXXXXXXXXXXX_freq"
```

Comment:
Are we pre-populating this?

Reply:
These were markers I used to test, and forgot to take them out before pushing.  Nice catch!